src/benchmark_register.cc

```diff
@@ -159,6 +159,7 @@ bool BenchmarkFamilies::FindBenchmarks(
         instance.use_manual_time = family->use_manual_time_;
         instance.complexity = family->complexity_;
         instance.complexity_lambda = family->complexity_lambda_;
+        instance.statistics = &family->statistics_;
```

Comment:
everything else here takes a copy rather than a reference (pointer). is it possible the lifetime of family is shorter than that of instance? should you take a copy (even though it'll be a memory/cpu hit)?